.github/workflows/unit-tests.yaml

```diff
@@ -40,13 +40,10 @@ jobs:
       - name: Install package dependencies.
         run: make install
 
-      - name: Generate OpenAPI spec for blocklist client
-        run: make generate-openapi-spec
+      - name: Run Basic tests
+        run: make test
```

Comment:
Right now the `make integration-tests` command runs the basic tests as well. We can either remove this or update the stanza in the yaml or update the command. I'm okay with either.

Reply:
I'll remove the test command then push it.